flashcards/backend/config/settings.py

```diff
@@ -174,13 +174,12 @@
     "DEFAULT_PARSER_CLASSES": [
         "rest_framework.parsers.JSONParser",
     ],
-    "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
 }
 
 # Simple JWT
 
 SIMPLE_JWT = {
-    "ACCESS_TOKEN_LIFETIME": timedelta(minutes=5),
+    "ACCESS_TOKEN_LIFETIME": timedelta(days=5),
```

Comment:
hmmmm JWT shouldn't be that long

Reply:
well access token to be precise

Reply:
Since authentication isn't part of this task I suggest returning so the initial version.  😅